tests/pkg_fatfs_vfs/main.c

```diff
@@ -286,7 +286,7 @@ static void test_create(void)
     ssize_t nw;
     print_test_result("test_create__mount", vfs_mount(&_test_vfs_mount) == 0);
 
-    fd = vfs_open(FULL_FNAME1, O_WRONLY | O_CREAT, 0);
+    fd = vfs_open(FULL_FNAME1, O_WRONLY | O_CREAT | O_TRUNC, 0);
```

Comment:
Don't change the test case, that's cheating :wink: 

We don't want to break existing code and there is a big difference between "Open File, Create if it doesn't exist." and "Open File, Create if it doesn't exist, otherwise clear it"

Reply:
Ah, sorry, that was a test i didn't want to push :/